frontend/cypress/integration/featureFiles/overview.feature

```diff
@@ -0,0 +1,79 @@
+Feature: Kiali Overview page
+
+  User opens the Overview page and see the demo "error-rates" namespaces
+
+  @focus
+  Scenario: See "alpha" and "beta" namespaces
+    Given user opens the overview page
+    Then user sees the "alpha" namespace
+    Then user sees the "beta" namespace
+
+  @focus
+  Scenario: Doesn't see a "bad" namespace
+    Given user opens the overview page
+    Then user doesn't see the "bad" namespace
```

Comment:
General comment where I'm just thinking out loud. There's some overhead involved with opening the overview page. I wonder if we'll want to limit the amount of separate scenarios in a test and combine some of these.

Reply:
Nope, testcase should be atomic. I would prefer high granularity with single (simple) assertions. We can [reduce steps in scenarios](https://cucumber.io/docs/gherkin/reference/#background) with few `Background:` steps, like:

```gherkin
Background: 
    Given user is at administrator perspective
    And user is at overview page

@overview-page
Scenario: See "alpha" and "beta" namespaces
    Then user sees the "alpha" namespace
    And user sees the "beta" namespace
```
@lucasponce we can save ~10 lines and get better readability 

Also chaining `Then` keyword does not make much sense, see example
```gherkin
Given url 'http://myhost.com/v1/cats&#39;'
And request { name: 'Billie' }
When method post
Then status 201
And match response == { id: '#notnull', name: 'Billie' }
```
Best practice is chain steps or conditions with `And` keyword.
If we want to early merge, I can request this modifications later

Reply:
> General comment where I'm just thinking out loud. There's some overhead involved with opening the overview page.

Well, +1 but I'd separate the "day 1" tasks from the "day 2" :).

For "day 1" I expect that we create a set of tests for all pages, today we have only work in progress for login and overview.

In the next iteration "day 2" we can refine and refactor the tests once we have more clarity about what's the best approach to maintain these tests.

Also, I expect that these tests will be part of the regular PR work, so any change in the UI will likely include a revision of the UI tests focused on the "feature" that is being tested.

I haven't completed all the scenarios I had in mind for the overview page (the actions on the namespaces are not yet covered), but I have interest to have a first version that may help to speed up the skeleton for other pages as well.